### ISSUE TITLE
Wormhole potion and wormhole mirror error

### ISSUE THREAD

**Issue by Darkere** [CursedFlames/BountifulBaubles#21]
Using either of the two dc's you with an Internal Server Error. I don't currently have access to the Log but will try to get it later. 

Haven't tested singleplayer. 
Pack: Enigmatica 4
bountifulbaubles-1.14.4-0.0.2
forge: 28.1.87

**Comment by Darkere** [comment]
Serverside Error: 
```[30Nov2019 20:57:44.850] [Server thread/WARN] [net.minecraft.network.NetworkSystem/]: Failed to handle packet for /84.142.221.107:62764
net.minecraft.crash.ReportedException: Ticking player
	at net.minecraft.entity.player.ServerPlayerEntity.func_71127_g(ServerPlayerEntity.java:428) ~[?:?]
	at net.minecraft.network.play.ServerPlayNetHandler.func_73660_a(ServerPlayNetHandler.java:183) ~[?:?]
	at net.minecraft.network.NetworkManager.func_74428_b(NetworkManager.java:228) ~[?:?]
	at net.minecraft.network.NetworkSystem.func_151269_c(NetworkSystem.java:135) [?:?]
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:846) [?:?]
	at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:324) [?:?]
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:764) [?:?]
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:622) [?:?]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_201]
Caused by: java.lang.NullPointerException: A given capability is null
	at java.util.Objects.requireNonNull(Unknown Source) ~[?:1.8.0_201]
	at org.cyclops.cyclopscore.modcompat.capabilities.DefaultCapabilityProvider.getCapability(DefaultCapabilityProvider.java:42) ~[?:1.5.5]
	at net.minecraftforge.common.capabilities.CapabilityDispatcher.getCapability(CapabilityDispatcher.java:102) ~[?:?]
	at net.minecraftforge.common.capabilities.CapabilityProvider.getCapability(CapabilityProvider.java:118) ~[?:?]
	at net.minecraft.entity.LivingEntity.getCapability(LivingEntity.java:2890) ~[?:?]
	at net.minecraft.entity.player.PlayerEntity.getCapability(PlayerEntity.java:2106) ~[?:?]
	at cursedflames.bountifulbaubles.common.wormhole.ContainerWormhole.<init>(ContainerWormhole.java:56) ~[?:?]
	at cursedflames.bountifulbaubles.common.item.items.ItemPotionWormhole$1.createMenu(ItemPotionWormhole.java:68) ~[?:?]
	at net.minecraftforge.fml.network.NetworkHooks.openGui(NetworkHooks.java:188) ~[?:?]
	at net.minecraftforge.fml.network.NetworkHooks.openGui(NetworkHooks.java:135) ~[?:?]
	at cursedflames.bountifulbaubles.common.item.items.ItemPotionWormhole.doWormhole(ItemPotionWormhole.java:77) ~[?:?]
	at cursedflames.bountifulbaubles.common.item.items.ItemPotionWormhole.func_77654_b(ItemPotionWormhole.java:85) ~[?:?]
	at net.minecraft.item.ItemStack.func_77950_b(ItemStack.java:196) ~[?:?]
	at net.minecraft.entity.LivingEntity.func_71036_o(LivingEntity.java:2570) ~[?:?]
	at net.minecraft.entity.player.ServerPlayerEntity.func_71036_o(ServerPlayerEntity.java:981) ~[?:?]
	at net.minecraft.entity.LivingEntity.func_184608_ct(LivingEntity.java:2462) ~[?:?]
	at net.minecraft.entity.LivingEntity.func_70071_h_(LivingEntity.java:1965) ~[?:?]
	at net.minecraft.entity.player.PlayerEntity.func_70071_h_(PlayerEntity.java:232) ~[?:?]
	at net.minecraft.entity.player.ServerPlayerEntity.func_71127_g(ServerPlayerEntity.java:365) ~[?:?]
	... 8 more
[30Nov2019 20:57:44.904] [Server thread/INFO] [net.minecraft.network.play.ServerPlayNetHandler/]: Darkere lost connection: Internal server error
[30Nov2019 20:57:44.904] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer/]: Darkere left the game```

**Comment by Darkere** [comment]
Interestingly a thing I did not notice the first time is that just right clicking with the potion tells me no tp targets available but drinking it disconnects.

**Comment by Hannah-GBS** [comment]
Seconding this issue. Would be great to see this working as intended as I'd be able to convince the server I play on to cut down on /tp "cheats"

**Comment by kil0byt3** [comment]
Also occurring on 1.12, see issue linked https://github.com/CalaMariGold/Rebirth-Of-The-Night/issues/134

**Comment by CursedFlames** [comment]
Should be fixed in 1.12 by #32, will investigate 1.14 shortly
